homeassistant/components/satel_integra/alarm_control_panel.py

```diff
@@ -75,6 +75,7 @@ def __init__(self, controller, name, arm_home_mode, partition_id):
         self._arm_home_mode = arm_home_mode
         self._partition_id = partition_id
         self._satel = controller
+        self._attr_unique_id = f"{partition_id}"
```

Comment:
```suggestion
        self._attr_unique_id = partition_id
```

Reply:
Also, I wonder if this is unique and a stable ID? What is the partition  ID exactly?

If I configure 2 instances of this integration, will those still be unique?

Reply:
Hmm, you might be right, those won't be unique across multiple instances. I'll see if I can use the host as part of the ID

Reply:
About the stable ID of the partition, this is basically a part of the configuration inside Satel Integra itself, a partition is a separate part of the alarm system that can be turned on/off independent from each other. Those IDs are actually set by the installer and are also linked to the actual wiring (at least in my case) to the control boards

Reply:
>  I'll see if I can use the host as part of the ID

That won't work, as the host part isn't considered stable.

> About the stable ID of the partition, this is basically a part of the configuration inside Satel Integra itself, a partition is a separate part of the alarm system that can be turned on/off independent from each other. 

Right, but if they are not unique, they cannot be used as the unique identifier, unless they are accompanied by something else that makes them unique. E.g., a serial number of the system.

Reply:
> That won't work, as the host part isn't considered stable.

I realized that after reading the developer documentation. I'm currently in the process of migrating the Satel Integra integration to be set up from a config flow, that way we can use that ID for the unique IDs.
Is it preferred to make 1 PR which includes both the config flow and the unique ID changes or is it better to split them up?

Reply:
The smaller the PRs the easier they are to review, the quicker things go :)